### ISSUE TITLE
NameError: name '__version__' is not defined

### ISSUE THREAD

**Issue by Jo-the-bat** [THLO/map#1]
Hello there ! I'm using Python 3.8.5 and it seem that there is a problem somewhere, i tried to install map and actually I have problems to install it: NameError: name '__version__' is not defined.

version.py:
` 
"""
This is the version file for the map project.

More information about map is available at https://github.com/THLO/map.
"""

__version__ = "1.2.0"

__version_text__ = 'Copyright (C) 2016 Thomas Locher\n\
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>.\n\
This is free software: you are free to change and redistribute it.\n\
There is NO WARRANTY, to the extent permitted by law.\n\n\
Written by Thomas Locher, see https://github.com/THLO/map.'

print(__version__) 

`

**It print 1.2.0**

**But**

setup.py:
`
try:
    from setuptools import setup
except ImportError:
    from distutils.core import setup

import os

def getVersion():
    with open(os.path.join(os.path.dirname(__file__),'map/version.py')) as f: exec(f.read())
    return __version__

setup(name = "map",
    version = "1.2.0",
    description = "map applies a command to files/folders at a certain path.",
    author = "Thomas Locher",
    author_email = "thamasta@gmx.ch",
    url = "https://github.com/THLO/map",
    download_url = "https://github.com/THLO/map/tarball/v."+getVersion(),
    packages = ['map'],
    scripts = ['map/map'],
    long_description = "The same functionality to apply a command to multiple files/folders\n\
can be achieved using for loops or find -exec or similar tools.\n\
The advantage of map is that it is easier and more convenient to use.\n\
More information on map can be found at https://github.com/THLO/map or by running map --help.",
    license = 'GNU General Public License v3 (GPLv3)',
    platforms = 'POSIX',
    classifiers = [
        'Development Status :: 5 - Production/Stable',
        'Environment :: Console',
        'Intended Audience :: End Users/Desktop',
        'Intended Audience :: Developers',
        'License :: OSI Approved :: GNU General Public License v3 (GPLv3)',
        'Operating System :: POSIX :: Linux',
        'Programming Language :: Python',
        'Topic :: Desktop Environment',
      ]
)

`

And it make the error 
Traceback (most recent call last):
  File "C:\<folders>", line 18, in <module>
    download_url = "https://github.com/THLO/map/tarball/v."+getVersion(),
  File "C:\<folders>", line 10, in getVersion
    return __version__
NameError: name '__version__' is not defined

Please, help me to resolve that.

**Comment by Lescurel** [comment]
Specify explicitly that the exec should append to the `globals` variable : 

```
def getVersion():
    with open(os.path.join(os.path.dirname(__file__),'map/version.py')) as f: 
        exec(f.read(), globals())
    return __version__

**Comment by THLO** [comment]
@Lescurel Thanks for proposing the fix!
The fix is now also in version 1.2.1, which I just uploaded.
